src/auth/entities/user.entity.ts

```
@@ -36,6 +36,7 @@ export class User {
 
   @Column()
   password: string
+  applications: any;
```

Biome (1.9.4)

[error] Duplicate class member name "applications"



(lint/suspicious/noDuplicateClassMembers)